packages/components/src/components/wordmark/wordmark-styles.ts

```diff
@@ -0,0 +1,53 @@
+import type { WordmarkSize } from './wordmark-utils';
+import type { Theme } from '../../types';
+import { getCss, isThemeDark } from '../../utils';
+import { addImportantToEachRule, getInsetJssStyle, getThemedColors, hostHiddenStyles } from '../../styles';
+import { filterLightPrimary, filterDarkPrimary } from '../../styles/color-filters';
+import { borderRadiusSmall, borderWidthBase } from '@porsche-design-system/utilities-v2/';
+
+export const getComponentCss = (size: WordmarkSize, theme: Theme): string => {
+  const isSizeFluid = size === 'small';
+  const { focusColor } = getThemedColors(theme);
+  const isDark = isThemeDark(theme);
+
+  return getCss({
+    '@global': {
+      ':host': {
+        position: 'relative',
+        display: 'inline-flex',
+        verticalAlign: 'top',
+        ...addImportantToEachRule({
+          outline: 0,
+          ...hostHiddenStyles,
+          boxSizing: 'content-box', // needed for custom clickarea to revert styles from Vue component
+          ...(isSizeFluid && { height: 'clamp(0.63rem, 0.42vw + 0.5rem, 1rem)' }),
```

Comment:
Don't we have a variable for this?

Reply:
We do not export this variable yet within utilities package. As discussed in private chat: I added a comment :)